pkg/apis/serving/fieldmask.go

```diff
@@ -161,6 +161,9 @@ func PodSpecMask(ctx context.Context, in *corev1.PodSpec) *corev1.PodSpec {
 	if cfg.Features.PodSpecNodeSelector != config.Disabled {
 		out.NodeSelector = in.NodeSelector
 	}
+	if cfg.Features.PodSpecRuntimeClassName != config.Disabled {
```

Comment:
worth adding a test for this, along same lines as [the one for PodSpecTolerations](https://github.com/knative/serving/blob/884f4ed3203907d9c9456a57e0257ddb62173933/pkg/apis/serving/k8s_validation_test.go#L586)?

Reply:
Done